### ISSUE TITLE
military comparison skips my own empire

### ISSUE THREAD

**Issue by Akuukis** [benreid24/Stellaru#87]
The Custom Chart tool silently skips my own empire at military comparison, and then fails to show anything (see screenshot).

The preset Comparison chart military view excludes my own empire.

![2021-10-03_184100](https://user-images.githubusercontent.com/7459419/135766059-b63ce0f9-f1a1-4734-8a2a-4c296bde1306.png)

**Comment by benreid24** [comment]
Good catch, thanks for the report. I'll take a look and see if I can figure out what's going on.
